apps/ensindexer/src/api/index.ts

```diff
@@ -18,19 +20,14 @@ import {
   buildGraphQLSchema as buildSubgraphGraphQLSchema,
   graphql as subgraphGraphQL,
 } from "@ensnode/ponder-subgraph";
-import {
-  addDocStringsToIntrospection,
-  extendWithBaseDefinitions,
-  generateTypeDocSet,
-} from "ponder-enrich-gql-docs-middleware";
 
 const {
```

Comment:
you may be planning on doing this in a future commit, and if so sorry for the eager review note, but it may read better if this destructuring were done during import like

```ts
import { requestedPluginNames, ... } from "@/config/app-config"
```

OR we should always

```ts
import config from "@/config/app-config"
```

and reference variables without destructuring like `config.requestedPluginNames` for clarity.

---

whichever pattern is preferred should be used exclusively, for consistence, in my opinion. personally i like the minimalism of the first option where the config values used in the file are stated and destructured in the import statement